tests/cucumber/steps/stepdefs.js

```diff
@@ -1560,6 +1568,43 @@ Given('mock http responses in {string} loaded from {string}', function (fileName
     this.indexerClient = new indexer.IndexerClient('', mockAlgodResponderHost, mockAlgodResponderPort, {});
 });
 
+Given('mock http responses in {string} loaded from {string} with status {int}.', function (fileName, jsonDirectory, status) {
+    setupMockServerForResponses(fileName, jsonDirectory, algodMockServerResponder);
+    setupMockServerForResponses(fileName, jsonDirectory, indexerMockServerResponder);
+    this.v2Client = new clientv2.AlgodClient('', mockAlgodResponderHost, mockAlgodResponderPort, {});
+    this.indexerClient = new indexer.IndexerClient('', mockAlgodResponderHost, mockAlgodResponderPort, {});
+    this.expectedMockResponseCode = status;
+});
+
+When('we make any {string} call to {string}.', async function (client, endpoint) {
+    try {
+        if (client == "algod") {
+            // endpoints are ignored by mock server, see setupMockServerForResponses
+            this.actualMockResponse = await this.v2Client.status().do();
+        } else if (client == "indexer") {
+            // endpoints are ignored by mock server, see setupMockServerForResponses
+            this.actualMockResponse = await this.indexerClient.makeHealthCheck().do();
+        } else {
+            throw Error("did not recognize desired client \"" + client + "\"");
+        }
+    } catch (err) {
+        if (this.expectedMockResponseCode == 200) {
+            throw err;
+        }
+        if (this.expectedMockResponseCode == 500) {
+            if (!err.toString().includes("Internal Server Error")) {
```

Comment:
Is there no way to get the actual error message?

Reply:
`err.toString()` is the actual error message, I think, unless I misunderstand?

Reply:
The error responses from the backend are designed to help the user, the error response is formatted like this:
```
{
  "data" : "oZSrqon3hT7qQqrHNUle",
  "message" : "oZSrqon3hT7qQqrHNUle"
}
```

Reply:
Hm, unfortunately `err.toString()` just gives `Internal Server Error`, maybe we file a ticket about that.